src/main/java/org/kiegroup/kogibot/listeners/pr/impl/PRFirstTimeContributorListener.java

```diff
@@ -0,0 +1,51 @@
+package org.kiegroup.kogibot.listeners.pr.impl;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import org.kiegroup.kogibot.config.KogibotConfiguration;
+import org.kiegroup.kogibot.listeners.pr.PRCommentConfigListener;
+import org.kiegroup.kogibot.util.Constants;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHRepository.Contributor;
+import org.kohsuke.github.GitHub;
+
+import io.quarkiverse.githubapp.ConfigFile;
+import io.quarkiverse.githubapp.event.PullRequest;
+
+public class PRFirstTimeContributorListener implements PRCommentConfigListener {
```

Comment:
as it is already inside the pr package, wouldn't be better to remove the PR prefix?

Reply:
Yes we could, gonna update that